pkg/cmd/operator/manager_controller.go

```diff
@@ -121,7 +121,7 @@ func (o *ManagerControllerOptions) Complete() error {
 	}
 
 	// TODO: Use https and wire certs.
-	managerClient, err := mermaidclient.NewClient("http://scylla-manager.scylla-manager.svc/api/v1", &http.Transport{})
+	managerClient, err := mermaidclient.NewClient("http://scylla-manager/api/v1", &http.Transport{})
```

Comment:
How about we search for a service with particular label in every namespace?

Reply:
the manager controller and manager are always in the same namespace though, right?

Reply:
yeah, namespace is the same. Although name can be overwriten in helm values file: https://github.com/scylladb/scylla-operator/blob/d2b52a604f4327107d8b1e2fa1dd7e0a7fb76d83/helm/scylla-manager/templates/_helpers.tpl#L15

Reply:
we should configurations for stuff that doesn't need it - there is no value in configuring resource names in a namespace we fully own

Reply:
> service with particular label in every namespace
sounds a bit like an overkill to search in every namespace (if there can be thousands) and the service labels are also configurable and not fixed

Reply:
having #667 lets just extract const values for these names in `naming/names.go` and use them here

Reply:
made it const